### DISCUSSION TITLE
How to use custom viewport size

### DISCUSSION THREAD

**Discussion by FRANKACQUAH** [GoogleChrome/lighthouse#14683] [Q&A]
Hi, How do I change the viewport of lighthouse CLI from the default Mobile to Desktop to Tablet view? 

Any help?

**Comment by adamraine** [comment]
From `lighthouse --help`:

```
--screenEmulation               Sets screen emulation parameters. See also --preset. Use --screenEmulation.disabled to disable. Otherwise set these 4 parameters individually: --screenEmulation.mobile --screenEmulation.width=360 --screenEmulation.height=640 --screenEmulation.deviceScale
                                      Factor=2
```

You will need to set the dimensions of your desired tablet yourself, we don't have a tablet preset.

**Reply by OZZlE** [reply]
Why can we not have an audit mode for tablet by default also? There's quite a few people using tablets, how are we developers supposed to check if our websites have good performance there??

I don't even understand your answer thb.. Can you start chrome command line with a flag somehow?? Is that the linux, mac or windows command? What comes before it? the .exe ? Only people working at Google can understand what you wrote I believe. #realitycheck

**Reply by adamraine** [reply]
> Why can we not have an audit mode for tablet by default also? There's quite a few people using tablets, how are we developers supposed to check if our websites have good performance there??

By changing the emulation settings as I described. If you want more information we have documentation on the subject:
https://github.com/GoogleChrome/lighthouse/blob/main/docs/emulation.md

Even if there is not a preset for your exact use case, the configuration options are available if you want to use them.

> I don't even understand your answer thb.. Can you start chrome command line with a flag somehow?? Is that the linux, mac or windows command? What comes before it? the .exe ? Only people working at Google can understand what you wrote I believe. #realitycheck

My answer uses the [Lighthouse CLI](https://github.com/GoogleChrome/lighthouse/blob/main/readme.md#using-the-node-cli) as that is what the original question was about.
